src/EnergyPlus/OutAirNodeManager.cc

```diff
@@ -390,19 +439,29 @@ namespace OutAirNodeManager {
 				Node( NodeNum ).OutAirDryBulb = OutDryBulbTempAt( Node( NodeNum ).Height );
 				Node( NodeNum ).OutAirWetBulb = OutWetBulbTempAt( Node( NodeNum ).Height );
 			}
+			Node( NodeNum ).OutAirWindSpeed = WindSpeed;
+			Node( NodeNum ).OutAirWindDir = WindDir;
 
-			if ( Node( NodeNum ).EMSOverrideOutAirDryBulb ) Node( NodeNum ).OutAirDryBulb = Node( NodeNum ).EMSValueForOutAirDryBulb;
+			// Set node data to local air node values if defined
+			if ( Node( NodeNum ).SchedOutAirDryBulb ) Node( NodeNum ).OutAirDryBulb = GetCurrentScheduleValue( Node( NodeNum ).OutAirDryBulbSchedNum );
+			if ( Node( NodeNum ).SchedOutAirWetBulb ) Node( NodeNum ).OutAirWetBulb = GetCurrentScheduleValue( Node( NodeNum ).OutAirWetBulbSchedNum );
+			if ( Node( NodeNum ).SchedOutAirWindSpeed ) Node( NodeNum ).OutAirWindSpeed = GetCurrentScheduleValue( Node( NodeNum ).OutAirWindSpeedSchedNum );
+			if ( Node( NodeNum ).SchedOutAirWindDir ) Node( NodeNum ).OutAirWindDir = GetCurrentScheduleValue( Node( NodeNum ).OutAirWindDirSchedNum );
 
-			if ( Node( NodeNum ).EMSOverrideOutAirWetBulb ) {
-				Node( NodeNum ).OutAirWetBulb = Node( NodeNum ).EMSValueForOutAirWetBulb;
-				Node( NodeNum ).HumRat = PsyWFnTdbTwbPb( Node( NodeNum ).OutAirDryBulb, Node( NodeNum ).OutAirWetBulb, OutBaroPress );
-				Node( NodeNum ).Enthalpy = PsyHFnTdbW( Node( NodeNum ).OutAirDryBulb, Node( NodeNum ).HumRat );
-			} else {
-				Node( NodeNum ).HumRat = OutHumRat;
-				Node( NodeNum ).Enthalpy = PsyHFnTdbW( Node( NodeNum ).OutAirDryBulb, OutHumRat );
-			}
+			// Set node data to EMS overwritten values if defined
+			if ( Node( NodeNum ).EMSOverrideOutAirDryBulb ) Node( NodeNum ).OutAirDryBulb = Node( NodeNum ).EMSValueForOutAirDryBulb;
+			if ( Node( NodeNum ).EMSOverrideOutAirWetBulb ) Node( NodeNum ).OutAirWetBulb = Node( NodeNum ).EMSValueForOutAirWetBulb;
+			if ( Node( NodeNum ).EMSOverrideOutAirWindSpeed ) Node( NodeNum ).OutAirWindSpeed = Node( NodeNum ).EMSValueForOutAirWindSpeed;
+			if ( Node( NodeNum ).EMSOverrideOutAirWindDir ) Node( NodeNum ).OutAirWindDir = Node( NodeNum ).EMSValueForOutAirWindDir;
 
 			Node( NodeNum ).Temp = Node( NodeNum ).OutAirDryBulb;
+			if ( Node( NodeNum ).SchedOutAirDryBulb || Node( NodeNum ).SchedOutAirWetBulb ) {
+				Node( NodeNum ).HumRat = PsyWFnTdbTwbPb( Node( NodeNum ).OutAirDryBulb, Node( NodeNum ).OutAirWetBulb, OutBaroPress );				
```

Comment:
@Myoldmopar What's the logic of EMS override value? I was trying to apply the same but can't find as well. The node level checking should be done after the EMS value also be override, right? So before HumRat and Enthalpy calculation?

```
			// Set node data to local air node values if defined
			if ( Node( NodeNum ).OutAirDryBulbSchedNum != 0 ) {
				Node( NodeNum ).OutAirDryBulb = GetCurrentScheduleValue( Node( NodeNum ).OutAirDryBulbSchedNum );
			}
			if ( Node( NodeNum ).OutAirWetBulbSchedNum != 0 ) {
				Node( NodeNum ).OutAirWetBulb = GetCurrentScheduleValue( Node( NodeNum ).OutAirWetBulbSchedNum );
			}
			if ( Node( NodeNum ).OutAirWindSpeedSchedNum != 0 ) {
				Node( NodeNum ).OutAirWindSpeed = GetCurrentScheduleValue( Node( NodeNum ).OutAirWindSpeedSchedNum );
			}
			if ( Node( NodeNum ).OutAirWindDirSchedNum != 0 ) {
				Node( NodeNum ).OutAirWindDir = GetCurrentScheduleValue( Node( NodeNum ).OutAirWindDirSchedNum );
			}

			// Set node data to EMS overwritten values if defined
			if ( Node( NodeNum ).EMSOverrideOutAirDryBulb ) Node( NodeNum ).OutAirDryBulb = Node( NodeNum ).EMSValueForOutAirDryBulb;
			if ( Node( NodeNum ).EMSOverrideOutAirWetBulb ) Node( NodeNum ).OutAirWetBulb = Node( NodeNum ).EMSValueForOutAirWetBulb;
			if ( Node( NodeNum ).EMSOverrideOutAirWindSpeed ) Node( NodeNum ).OutAirWindSpeed = Node( NodeNum ).EMSValueForOutAirWindSpeed;
			if ( Node( NodeNum ).EMSOverrideOutAirWindDir ) Node( NodeNum ).OutAirWindDir = Node( NodeNum ).EMSValueForOutAirWindDir;
```